S1API/Products/ProductDefinition.cs

```diff
@@ -26,8 +27,11 @@ public class ProductDefinition : ItemDefinition
         /// INTERNAL: Creates a product definition from the in-game product definition.
         /// </summary>
         /// <param name="productDefinition"></param>
-        internal ProductDefinition(S1Product.ProductDefinition productDefinition) : base(productDefinition) { }
-
+        #if IL2CPPBEPINEX || IL2CPPMELON
+        internal ProductDefinition(Il2CppScheduleOne.ItemFramework.ItemDefinition productDefinition) : base(productDefinition) { }
+#else
+        internal ProductDefinition(ScheduleOne.ItemFramework.ItemDefinition productDefinition) : base(productDefinition) { }
+        #endif
```

Comment:
```suggestion
#if IL2CPPBEPINEX || IL2CPPMELON
        internal ProductDefinition(Il2CppScheduleOne.ItemFramework.ItemDefinition productDefinition) : base(productDefinition) { }
#else
        internal ProductDefinition(ScheduleOne.ItemFramework.ItemDefinition productDefinition) : base(productDefinition) { }
#endif
```
Be careful of code styling here